pool/entities.go

```diff
@@ -0,0 +1,256 @@
+package pool
+
+type ID string
+
+type Name string
+
+// I don't know i have to bring Peer Available storage and other metric inside Peer
+type Peer string
+
+type ReplicationFactor int8
```

Comment:
`uint8`? and why 8 bit?

Reply:
Maybe too small. I will change it to int.